server/sonar-server/src/main/java/org/sonar/server/computation/task/projectanalysis/formula/CreateMeasureContext.java

```diff
@@ -40,6 +40,18 @@
 
   /**
    * The periods for which variations of the measure can be created.
+   * @deprecated as only one period is now available. Use {@link #getPeriod()} instead
    */
+  @Deprecated
   List<Period> getPeriods();
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)